src/main/java/net/pms/util/FileUtil.java

```diff
@@ -1247,7 +1248,7 @@ public static String[] getFileNameMetadata(String filename, String absolutePath)
 						tvEpisodeName = convertFormattedNameToTitleCase(tvEpisodeName);
 					}
 				} else {
-					showNameIndex = indexOf(Pattern.compile(showNameIndexMatcher), formattedName);
+					showNameIndex = indexOf(Pattern.compile(SHOW_NAME_INDEX_MATCHER), formattedName);
```

Comment:
`Pattern.compile(SHOW_NAME_INDEX_MATCHER)`

this one can also be statically compiled
eg:
`private static final Pattern SHOW_NAME_INDEX_FALLBACK_PATTERN = Pattern.compile(SHOW_NAME_INDEX_MATCHER);`

then

`showNameIndex = indexOf(SHOW_NAME_INDEX_FALLBACK_PATTERN, formattedName);`